src/charts/table/transform.ts

```diff
@@ -0,0 +1,19 @@
+import { TableChartSettings } from '@keen.io/charts';
+import { WidgetTransform } from '../../types';
+
+const transform: WidgetTransform<TableChartSettings> = {
```

Comment:
The `table` transformation should be added to `serializeInputSettings` function (switch - case block).